docs/docs/examples/property_graph/property_graph_advanced.ipynb

```diff
@@ -0,0 +1,345 @@
+{
```

Comment:
<p>nit: </p><p> </p><ul><li>make title more specific to the topic (e.g. "Advanced Property Graph *Construction [with neo4j]" or "Property Graph Construction with Predefined Schemas"). Seems like the Ollama part is nice but secondary. </li><li>add more details in the intro explaining what the purpose of nb is (e.g. we show you how to build a property graph w.r.t a user-defined schema of allowed entities and relationships) </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/13747/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>